lib/github/markups.rb

```diff
@@ -53,4 +53,4 @@
 
 command(::GitHub::Markups::MARKUP_POD, :pod2html, /pod/, ["Pod"], "pod")
 
-command(::GitHub::Markups::MARKUP_MANPAGE, :man2html, /1|2|3|4|5|6|7|8|man|mdoc/, ["Roff"], "manpage")
+command(::GitHub::Markups::MARKUP_MANPAGE, :man2html, /([1-9n][a-z]*|man|mdoc)(\.in)?/, ["Roff"], "manpage")
```

Comment:
If man pages in section 0, e.g. the POSIX string.h(0p), aren't going to be supported then can that please be stated so I can stop pointing out the regexp doesn't permit them.

Reply:
Maybe I misunderstood; I thought that pages in section 0 were plain-text and thus did not require formatting?

Reply:
[Nope.](https://packages.gentoo.org/packages/sys-apps/man-pages-posix)

Reply:
@Alhadis Is that just `0p` then, or should I treat `0` as any other section? I swore someone told me the `.0` files were pre-formatted...

Reply:
see @ischwarze comment https://github.com/github/markup/pull/1196#discussion_r214520839
> *.0 -- those are preformatted manual pages, essentially *.txt files; consequently, they rarely occur in source trees, but still, manual page formatters cannot handle them, you simply read them with a pager like more(1) or less(1)

Reply:
I've added `0[a-z]+` to the regex; hopefully that's an acceptable compromise.

Reply:
Yep. Sometimes called "catman pages", after the [`catman(1)`](https://www.freebsd.org/cgi/man.cgi?query=catman&sektion=1&manpath=FreeBSD+5.0-current) program which generates preformatted output for terminal display. So named because you could run `cat` on its output to see the formatted manpage.

The program was really only relevant in the old days when processors were limited in their power, and formatting a man-page was a potentially costly procedure.

Nowadays, processors are so powerful we hardly notice the time spent formatting the Roff markup, and having `catman` pages lingering around can actually be problematic if the manpage's source is needed...

Reply:
Perhaps @ischwarze can clarify where they occur when he has the time. I've not noticed a `.0' suffix being used for catman pages. catman(8) here says, regarding the sections to search, ‘The default is "1 n l 8 3 0 2 5 4 9 6 7"’, note the 0, and so man-db, where it's from, doesn't require a letter suffix and treats 0 as just another section for source needing formatting.

Reply:
I've added `0[a-z]+` to the regex; hopefully that's an acceptable compromise.

Reply:
Erf, didn't mean to comment twice; I'll leave it as until @ischwarze can weigh in  but based on catman(8) I'm leaning towards just treating .0 as a regular section instead of special casing it like it is now